packages/@aws-cdk/aws-eks/lib/cluster-resource-handler/common.ts

```diff
@@ -14,6 +14,22 @@ export interface EksUpdateId {
 
 export type ResourceEvent = AWSLambda.CloudFormationCustomResourceEvent & EksUpdateId;
 
+/**
+ * Decodes encoded true/false values
+ */
+function decodeBooleans(object: object) {
```

Comment:
Why is this needed if we have this in the custom resource library?

Reply:
you are right. removed